### PR TITLE
refactor(weaver): prefix cacti to all weaver packages

### PR DESCRIPTION
**Pull request by sandeepnRES**
Set publish target to cacti

- Update relay, drivers, iin-agent image registry to hyperledger/cacti
- Update npm packages (protos-js and sdks) registry to cacti
- Update maven packages registry to cacti
- Rename sdks/besu/interoperation-node-sdk to sdks/besu/node
- Rename modules with following convention `<module-type>-<dlt-platform>`:
    - For SDKs: `sdk.corda`, `sdk-fabric`, `sdk-besu`
    - For drivers: `driver-corda`, `driver-fabric`
    - For interoperation modules: `imodule.corda`
- Change all packages version to v2.0.0-alpha-prerelease

### CODE REVIEW

**Comment by sandeepnRES** [comment]
Large number of files are modified because of package name and registry changes to all weaver modules, which now targets to cacti. (Its mostly a find and replace)

**Review by petermetz — APPROVED** [review]
> Large number of files are modified because of package name and registry changes to all weaver modules, which now targets to cacti. (Its mostly a find and replace)

@sandeepnRES That's okay for cases when the **only** thing you do is these administrative changes (so I'm personally OK with this). 
What I try to always avoid is having PRs that are doing some actual change in the logic/features AND mixing in huge diffs at the same time (because then if I go back later to look at that diff trying to figure out what was changed  it is very hard because the large diff of the administrative changes are adding noise to the signal of the little diff changes of logic).
Just a long way of saying that I think that this LGTM. Thank you!

**Review by izuru0 — APPROVED** [review]
LGTM

File: .github/workflows/test_weaver-data-sharing.yaml [hunk 1]
```diff
@@ -450,34 +450,34 @@ jobs:
...
-          sed -i "s#^DOCKER_IMAGE_NAME=.*#DOCKER_IMAGE_NAME=weaver-relay-server#g" docker/testnet-envs/.env.corda2
+          sed -i "s#^DOCKER_IMAGE_NAME=.*#DOCKER_IMAGE_NAME=cacti-weaver-relay-server#g" docker/testnet-envs/.env.corda2
           make start-server COMPOSE_ARG='--env-file docker/testnet-envs/.env.corda2'
         working-directory: weaver/core/relay
 
       # FABRIC DRIVER
       - name: Setup Fabric Driver .env
         run: |
-          sed -i "s#^DOCKER_IMAGE_NAME=.*#DOCKER_IMAGE_NAME=weaver-fabric-driver#g" docker-testnet-envs/.env.n1
+          sed -i "s#^DOCKER_IMAGE_NAME=.*#DOCKER_IMAGE_NAME=cacti-weaver-fabric-driver#g" docker-testnet-envs/.env.n1
```
Comment (VRamakrishna) [inline .github/workflows/test_weaver-data-sharing.yaml:478]: Now that we are publishing these packages afresh in a new repository, I think we ought to name them in a consistent way, with some sense of hierarchy. So I'd suggest naming this to `cacti-weaver-driver-fabric`.

File: .github/workflows/test_weaver-data-sharing.yaml [hunk 2]
```diff
@@ -525,13 +525,13 @@ jobs:
       # CORDA DRIVER
       - name: Start Corda Driver
         run: |
-          sed -i "s#^DOCKER_IMAGE_NAME=.*#DOCKER_IMAGE_NAME=weaver-corda-driver#g" docker-testnet-envs/.env.corda
+          sed -i "s#^DOCKER_IMAGE_NAME=.*#DOCKER_IMAGE_NAME=cacti-weaver-corda-driver#g" docker-testnet-envs/.env.corda
```
Comment (VRamakrishna) [inline .github/workflows/test_weaver-data-sharing.yaml:528]: Now that we are publishing these packages afresh in a new repository, I think we ought to name them in a consistent way, with some sense of hierarchy. So I'd suggest naming this to `cacti-weaver-driver-corda`.

File: weaver/core/drivers/corda-driver/README.md [hunk 3]
```diff
@@ -120,7 +120,7 @@ To push image to github container registry:
     - `DRIVER_RPC_PASSWORD`: Password for the above RPC user.
     - `EXTERNAL_NETWORK`: Name of the docker network in which the Corda containers are deployed.
     - `DOCKER_IMAGE_NAME`: _Keep this unchanged_.
-    - `DOCKER_TAG`: Refer here for the image tags available: [weaver-corda-driver](https://github.com/hyperledger-labs/weaver-dlt-interoperability/pkgs/container/weaver-corda-driver)
+    - `DOCKER_TAG`: Refer here for the image tags available: [cacti-weaver-corda-driver](https://github.com/hyperledger/cacti/pkgs/container/cacti-weaver-corda-driver)
```
Comment (VRamakrishna) [inline weaver/core/drivers/corda-driver/README.md:123]: If we change the package name to `cacti-weaver-driver-corda`, make sure to edit this too.

File: weaver/core/drivers/corda-driver/docker-testnet-envs/.env.corda [hunk 4]
```diff
@@ -2,7 +2,7 @@ NETWORK_NAME=Corda_Network
 DRIVER_PORT=9099
 DRIVER_RPC_USERNAME=driverUser1
 DRIVER_RPC_PASSWORD=test
-DOCKER_IMAGE_NAME=ghcr.io/hyperledger-labs/weaver-corda-driver
+DOCKER_IMAGE_NAME=ghcr.io/hyperledger/cacti-weaver-corda-driver
```
Comment (VRamakrishna) [inline weaver/core/drivers/corda-driver/docker-testnet-envs/.env.corda:5]: Same comment as earlier.

File: weaver/core/drivers/fabric-driver/package-local.json [hunk 5]
```diff
@@ -19,8 +19,8 @@
     "author": "",
     "license": "ISC",
     "dependencies": {
-        "@hyperledger-labs/weaver-protos-js": "file:./protos-js",
-        "@hyperledger-labs/weaver-fabric-interop-sdk": "file:./weaver-fabric-interop-sdk",
+        "@hyperledger/cacti-weaver-protos-js": "file:./protos-js",
+        "@hyperledger/cacti-weaver-fabric-sdk": "file:./weaver-fabric-interop-sdk",
```
Comment (VRamakrishna) [inline weaver/core/drivers/fabric-driver/package-local.json:23]: Suggestion: name this package `cacti-weaver-sdk-fabric`.

File: weaver/core/drivers/fabric-driver/tsconfig.json [hunk 6]
```diff
@@ -15,7 +15,7 @@
     // "sourceMap": true,                     /* Generates corresponding '.map' file. */
     // "outFile": "./",                       /* Concatenate and emit output to single file. */
     "outDir": "./out",                        /* Redirect output structure to the directory. */
-    // "rootDir": "./",                       /* Specify the root directory of input files. Use to control the output directory structure with --outDir. */
+    // "rootDir": "./",                          /* Specify the root directory of input files. Use to control the output directory structure with --outDir. */
```
Comment (VRamakrishna) [inline weaver/core/drivers/fabric-driver/tsconfig.json:18]: Unnecessary whitespace?

Comment (sandeepnRES) [reply weaver/core/drivers/fabric-driver/tsconfig.json:18]: Actually was doing something with `rootDir` and reverted it, the whitespace got left, will remove it

File: weaver/core/drivers/corda-driver/README.md [hunk 7]
```diff
@@ -1,9 +1,9 @@
-<!--
+driver-corda<!--
  Copyright IBM Corp. All Rights Reserved.
 
  SPDX-License-Identifier: CC-BY-4.0
  -->
-# corda-driver
+# driver-corda
```
Comment (VRamakrishna) [inline weaver/core/drivers/corda-driver/README.md:6]: This is the title, so let's call this `Corda Driver` and not use the package name.

File: weaver/core/drivers/fabric-driver/docker-compose.yml [hunk 8]
```diff
@@ -6,15 +6,15 @@
...
-    container_name: fabric-driver-${NETWORK_NAME}
+    container_name: driver-fabric-${NETWORK_NAME}
     environment:
       - NETWORK_NAME=${NETWORK_NAME}
       - RELAY_ENDPOINT=${RELAY_ENDPOINT}
       - DRIVER_ENDPOINT=0.0.0.0:${DRIVER_PORT}
-      - CONNECTION_PROFILE=/fabric-driver/ccp.json
-      - DRIVER_CONFIG=/fabric-driver/config.json
+      - CONNECTION_PROFILE=//driver/fabric/ccp.json
+      - DRIVER_CONFIG=//driver/fabric/config.json
```
Comment (VRamakrishna) [inline weaver/core/drivers/fabric-driver/docker-compose.yml:17]: Unnecessary extra `/` in both lines above.

File: weaver/core/drivers/fabric-driver/docker-compose.yml [hunk 9]
```diff
@@ -29,10 +29,10 @@ services:
...
       - MONITOR_SYNC_PERIOD=${MONITOR_SYNC_PERIOD}
     volumes:
-      - ${CONNECTION_PROFILE}:/fabric-driver/ccp.json
-      - ${DRIVER_CONFIG}:/fabric-driver/config.json
-      - ./wallet-${NETWORK_NAME}:/fabric-driver/wallet-${NETWORK_NAME}
-      - ${TLS_CREDENTIALS_DIR}:/fabric-driver/credentials
+      - ${CONNECTION_PROFILE}://driver/fabric/ccp.json
+      - ${DRIVER_CONFIG}://driver/fabric/config.json
+      - ./wallet-${NETWORK_NAME}://driver/fabric/wallet-${NETWORK_NAME}
+      - ${TLS_CREDENTIALS_DIR}://driver/fabric/credentials
```
Comment (VRamakrishna) [inline weaver/core/drivers/fabric-driver/docker-compose.yml:35]: Unnecessary extra `/` in 4 lines above.

File: weaver/core/drivers/fabric-driver/docker-testnet-envs/.env.n2.tls [hunk 10]
```diff
@@ -4,14 +4,14 @@ RELAY_ENDPOINT=relay-network2:9083
...
 INTEROP_CHAINCODE=interop
-DOCKER_IMAGE_NAME=ghcr.io/hyperledger-labs/weaver-fabric-driver
+DOCKER_IMAGE_NAME=ghcr.io/hyperledger/cacti-weaver-driver-fabric
 DOCKER_TAG=latest
 EXTERNAL_NETWORK=network2_net
 COMPOSE_PROJECT_NAME=network2
 COMPOSE_PROJECT_NETWORK=net
 RELAY_TLS=true
-RELAY_TLSCA_CERT_PATH=/fabric-driver/credentials/docker/ca-cert.pem
+RELAY_TLSCA_CERT_PATH=//driver/fabric/credentials/docker/ca-cert.pem
```
Comment (VRamakrishna) [inline weaver/core/drivers/fabric-driver/docker-testnet-envs/.env.n2.tls:13]: Extra `/`

File: weaver/core/drivers/fabric-driver/docker-testnet-envs/.env.n2.tls [hunk 11]
```diff
@@ -4,14 +4,14 @@ RELAY_ENDPOINT=relay-network2:9083
...
 DRIVER_TLS=true
-DRIVER_TLS_CERT_PATH=/fabric-driver/credentials/docker/fabric-driver-network2-cert.pem
-DRIVER_TLS_KEY_PATH=/fabric-driver/credentials/docker/fabric-driver-network2-key.pem
+DRIVER_TLS_CERT_PATH=//driver/fabric/credentials/docker/fabric-driver-network2-cert.pem
+DRIVER_TLS_KEY_PATH=//driver/fabric/credentials/docker/fabric-driver-network2-key.pem
```
Comment (VRamakrishna) [inline weaver/core/drivers/fabric-driver/docker-testnet-envs/.env.n2.tls:16]: Extra `/`

File: weaver/core/identity-management/iin-agent/package.json [hunk 12]
```diff
@@ -19,8 +19,8 @@
     "author": "",
     "license": "ISC",
     "dependencies": {
-        "@hyperledger-labs/weaver-protos-js": "~1.5.0",
-        "@hyperledger-labs/weaver-fabric-interop-sdk": "~1.5.0",
+        "@hyperledger/cacti-weaver-protos-js": "file:./protos-js",
+        "@hyperledger/cacti-weaver-sdk-fabric": "file:./cacti-weaver-sdk-fabric",
```
Comment (VRamakrishna) [inline weaver/core/identity-management/iin-agent/package.json:23]: Is this a temporary change?

File: weaver/core/network/corda-interop-app/makefile [hunk 13]
```diff
@@ -27,13 +27,8 @@ publish-contracts: github.properties
 test:
 	./gradlew test
 
-.PHONY: clean-local
-clean-local:
-	./gradlew clean
-	rm -rf fabric-protos proto || true
-	rm -rf .gradle
-
```
Comment (VRamakrishna) [inline weaver/core/network/corda-interop-app/makefile:35]: Why remove this?

File: weaver/core/network/corda-interop-app/makefile [hunk 14]
```diff
@@ -27,13 +27,8 @@ publish-contracts: github.properties
...
 .PHONY: clean
-clean:	github.properties
```
Comment (VRamakrishna) [inline weaver/core/network/corda-interop-app/makefile:37]: Why remove this?

File: weaver/core/network/fabric-interop-cc/makefile [hunk 15]
```diff
@@ -1,6 +1,6 @@
-DOCKER_IMAGE_NAME ?= weaver-fabric-cc
+DOCKER_IMAGE_NAME ?= cacti-weaver-imodule-fabric-cc
```
Comment (VRamakrishna) [inline weaver/core/network/fabric-interop-cc/makefile:1]: Remove the `-cc`.

File: weaver/samples/besu/besu-cli/tsconfig.json [hunk 16]
```diff
@@ -21,6 +21,7 @@
     "strict": false,
     "target": "es5",
     "declaration": true,
+    // "rootDir": "./",
```
Comment (VRamakrishna) [inline weaver/samples/besu/besu-cli/tsconfig.json:24]: Remove

File: weaver/core/network/corda-interop-app/makefile (hunk 13 above)
Comment (sandeepnRES) [reply weaver/core/network/corda-interop-app/makefile:35]: both `clean-local` and `clean` were doing same thing, so keeping only one `clean` in makefile

File: weaver/core/network/corda-interop-app/makefile (hunk 14 above)
Comment (sandeepnRES) [reply weaver/core/network/corda-interop-app/makefile:37]: same reason

File: weaver/core/identity-management/iin-agent/package.json (hunk 12 above)
Comment (sandeepnRES) [reply weaver/core/identity-management/iin-agent/package.json:23]: oh sorry may be during build the `local` file replaced original one.

File: weaver/sdks/besu/node/README.md [hunk 17]
```diff
@@ -3,7 +3,7 @@
 
  SPDX-License-Identifier: CC-BY-4.0
  -->
-# weaver-besu-interop-sdk
+# cacti-weaver-sdk-besu
```
Comment (VRamakrishna) [inline weaver/sdks/besu/node/README.md:6]: This is a title, so replace with `Cacti - Weaver Besu SDK`.

File: weaver/sdks/besu/node/tsconfig.json [hunk 18]
```diff
@@ -4,6 +4,7 @@
         "allowJs": true,
         "target": "es5",
         "esModuleInterop": true,
+        // "rootDir": "./",
```
Comment (VRamakrishna) [inline weaver/sdks/besu/node/tsconfig.json:7]: Remove

File: weaver/sdks/fabric/interoperation-node-sdk/README.md [hunk 19]
```diff
@@ -3,7 +3,7 @@
 
  SPDX-License-Identifier: CC-BY-4.0
  -->
-# weaver-fabric-interop-sdk
+# cacti-weaver-sdk-fabric
```
Comment (VRamakrishna) [inline weaver/sdks/fabric/interoperation-node-sdk/README.md:6]: This is a title, so replace with `Cacti - Weaver Fabric SDK`.

File: weaver/sdks/fabric/interoperation-node-sdk/tsconfig.json [hunk 20]
```diff
@@ -4,6 +4,7 @@
         "allowJs": true,
         "target": "es5",
         "esModuleInterop": true,
+        // "rootDir": "./",
```
Comment (VRamakrishna) [inline weaver/sdks/fabric/interoperation-node-sdk/tsconfig.json:7]: Remove

File: weaver/samples/besu/besu-cli/tsconfig.json (hunk 16 above)
Comment (sandeepnRES) [reply weaver/samples/besu/besu-cli/tsconfig.json:24]: I'll remove this, actually there's a small issue, there's a global `node_modules` folder when we clone the `cacti` repo, and modules from that conflicts with weaver node packages, this `rootDir` kinds of fixes it but sometimes create other issues. So first I added it and then commented it out.

Comment (VRamakrishna) [reply weaver/samples/besu/besu-cli/tsconfig.json:24]: OK, remove it for now and open an issue to track this.

File: weaver/core/network/corda-interop-app/makefile (hunk 13 above)
Comment (sandeepnRES) [reply weaver/core/network/corda-interop-app/makefile:35]: Also one point, since `protos` are now built into `common/protos-java-kt`, these folders `fabric-protos` and `protos` are never created in this directory. (should have changed this long ago, but missed it)

**Review by VRamakrishna — APPROVED** [review]
